src/ui/components/SecondaryToolbox/index.tsx

```diff
@@ -70,6 +74,7 @@ const PanelButtons: FC<PanelButtonsProps> = ({
   hasReduxAnnotations,
   toolboxLayout,
   hasGraphics,
```

Comment:
I bet we can also get rid of `hasGraphics`

Reply:
Good call.